egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -424,6 +424,19 @@ def prepare_initial_acoustic_model(dir, run_opts, srand=-1):
     common_train_lib.prepare_initial_network(dir, run_opts,
                                              srand=srand)
 
+    # edits 0.raw using edits.config before adding transition model.
```

Comment:
Add an example for edits.config here and explain that it is used for transfer learning.